linux/LinuxProcess.c

```diff
@@ -106,11 +103,11 @@ const ProcessFieldData Process_fields[LAST_PROCESSFIELD] = {
  * happens on what is displayed - whether comm, full path, basename, etc.. So
  * this follows LinuxProcess_writeField(COMM) and LinuxProcess_writeCommand */
```

Comment:
This comment is out-of-date now - the reference to LinuxProcess_writeCommand should be Process_writeCommand now.

Reply:
Okay, oversight. Already fixed another instance and hoped I caught 'em all …